.github/workflows/build_and_test.yml

```diff
@@ -227,8 +227,9 @@ jobs:
         restore-keys: |
           ${{ matrix.java }}-${{ matrix.hadoop }}-coursier-
     - name: Install Java ${{ matrix.java }}
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v3
       with:
+        distribution: zulu
```

Comment:
Default to temurin maybe? I thought that was the more 'standard' one but I don't know

Reply:
Yes, `temurin` is more 'standard', zulu is default used in v1, so I kept it same. But I also think `temurin` no problem. Let's see others idea.

Reply:
Yeah, I think it;s fine to switch to `temurin`.